apps/cow-fi/components/SearchBar.tsx

```diff
@@ -91,15 +93,63 @@ const SearchResults = styled.div`
 const SearchResultsInner = styled.div`
```

Comment:
Also, since we are getting more styled elements here, would be great to move them into `styledEl.ts`

Reply:
Addressed.